tutorials/learnshell.org/de/classTutorial.md

```diff
@@ -0,0 +1,30 @@
+Tutorial: You will need to know basic shell scripting or any other programming language to solve this. You can use whichever loop you like. For, If, While etc...
```

Comment:
@Narayan51 please look at how to write tutorials using the Markdown syntax provided in the main directory and resubmit.